client/app/pages/dashboards/dashboard.js

```diff
@@ -172,7 +171,8 @@ function DashboardCtrl(
   };
 
   const renderDashboard = (dashboard, force) => {
-    Title.set(dashboard.name);
+    // TODO: Title should be updated after editing as well
```

Comment:
2a2bfb81c66bdfb30cc1727a72dc7ec292a9aeca

Reply:
Yes, I saw it few minutes ago; this comment I added this morning :grin: Let it stay here, anyway we'll remove this file soon